src/femr/labelers/benchmarks.py

```diff
@@ -0,0 +1,741 @@
+                value = value
+            else:
+                raise ValueError(f"Unknown unit: {unit}")
+        else:
+            raise ValueError(f"Unknown unit: {unit}")
+        if value < 3:
+            return "severe"
+        elif value < 3.5:
+            return "moderate"
+        elif value <= 3.9:
+            return "mild"
+        return "normal"
+
+
+class HyponatremiaInstantLabValueLabeler(InstantLabValueLabeler):
+    """lab-based definition for hyponatremia based on blood sodium concentration (mmol/L).
+    Thresholds: mild (<=135),moderate(<130),severe(<125), and abnormal range."""
+
+    original_omop_concept_codes = ["LOINC/LG11363-5", "LOINC/2951-2", "LOINC/2947-0"]
+
+    def value_to_label(self, raw_value: str, unit: Optional[str]) -> str:
+        if raw_value.lower() in ["normal", "adequate"]:
+            return "normal"
+        value = float(raw_value)
+        if value < 125:
+            return "severe"
+        elif value < 130:
+            return "moderate"
+        elif value <= 135:
+            return "mild"
+        return "normal"
+
+
+class AnemiaInstantLabValueLabeler(InstantLabValueLabeler):
+    """lab-based definition for anemia based on hemoglobin levels (g/L).
+    Thresholds: mild(<120),moderate(<110),severe(<70), and reference range"""
+
+    original_omop_concept_codes = [
+        "LOINC/LP392452-1",
+    ]
+
+    def value_to_label(self, raw_value: str, unit: Optional[str]) -> str:
+        if raw_value.lower() in ["normal", "adequate"]:
+            return "normal"
+        value = float(raw_value)
+        if unit is not None:
+            unit = unit.lower()
+            if unit.startswith("g/dl"):
+                # g / dL
+                # Original OMOP concept ID: 8713
+                # NOTE: This weird *10 / 100 is how Lawrence did it
+                value = value * 10
+            elif unit.startswith("mg/dl"):
+                # mg / dL (divide by 1000 to get g/dL)
+                # Original OMOP concept ID: 8840
+                # NOTE: This weird *10 / 100 is how Lawrence did it
+                value = value / 100
+            elif unit.startswith("g/l"):
+                value = value
+            else:
+                raise ValueError(f"Unknown unit: {unit}")
+        else:
+            raise ValueError(f"Unknown unit: {unit}")
+        if value < 70:
+            return "severe"
+        elif value < 110:
+            return "moderate"
+        elif value < 120:
+            return "mild"
+        return "normal"
+
+
+##########################################################
+##########################################################
+# First Diagnosis Tasks
+# See: https://github.com/som-shahlab/few_shot_ehr/tree/main
+#
+# Citation: Few shot EHR benchmark (ours)
+##########################################################
+##########################################################
+
+
+class FirstDiagnosisTimeHorizonCodeLabeler(Labeler):
```

Comment:
Why isn't this a subclass of the TimeHorizonLabeler?

Reply:
Good point -- fixed!